docs/topic_guides/evaluation/edge_metrics.md

```diff
@@ -0,0 +1,198 @@
+This guide is intended to be a reference guide for Edge Metrics used throughout Splink. It will build up from basic principles into more complex metrics.
+
+!!! note
+    All of these metrics are dependent on having a "ground truth" to compare against. This is generally provided by **Clerical Labelling** (i.e. labels created by a human). For more on how to generate this ground truth (and the impact that can have on Edge Metrics), check out the [Clerical Labelling Topic Guide](./labelling.md).
+
+## The Basics
+
+Any Edge (Link) within a Splink model will fall into one of four categories:
+
+### True Positive
+
+Also known as: True Link
+
+A True Positive is a case where a Splink model correctly predicts a match between two records.
+
+### True Negative
+
+Also known as: True Non-link
+
+A True Negative is a case where a Splink model correctly predicts a non-match between two records.
+
+### False Positive
+
+Also known as: False Link, Type I Error
+
+A False Positive is a case where a Splink model incorrectly predicts a match between two records, when they are actually a non-match.
+
+
+### False Negative
+
+Also known as: False Non-link, Missed Link, Type II Error
+
+A False Negative is a case where a Splink model incorrectly predicts a non-match between two records, when they are actually a match.
+
+
+### Confusion Matrix
+These can be summarised in a Confusion Matrix
+
+![](./image/confusion_matrix.drawio.png){:style="width:600px"}
+
+In a perfect model there would be no False Positives or False Negatives (i.e. FP = 0 and FN = 0).
+
+??? info "Confusion Matrix in Splink:"
+
+    - [confusion_matrix_from_labels_table](../../charts/confusion_matrix_from_labels_table.ipynb)
+
+## Metrics for Linkage
+
+The confusion matrix shows **counts** of each link type, but we are generally more interested in **proportions**. I.e. what percentage of the time does the model get the answer right?
+
+### Accuracy
+
+The simplest metric is 
+
+$$\textsf{Accuracy} = \frac{\textsf{True Positives}+\textsf{True Negatives}}{\textsf{All Predictions}}$$
+
+This measures the proportion of correct classifications (of any kind). This may be useful for balanced data but high accuracy can be achieved by simply assuming the majority class for highly imbalanced data (e.g. assuming non-matches).
+
+??? info "Accuracy in Splink:"
+
+    - [accuracy_chart_from_labels_table](../../charts/accuracy_chart_from_labels_table.ipynb)
```

Comment:
As above, maybe add a short intro?

Reply:
Yea you are right, all of these should have some additional commentary

Reply:
done